azurerm_container_registry - deprecating the georeplication_locations property in favour of the georeplications property (#11200)
terraform-providers/terraform-provider-azurerm: website/docs/r/container_registry.html.markdown

* `tags` - (Optional) A mapping of tags to assign to the resource.

> * `georeplication_locations` - (Optional / Deprecated in favor of georeplications) A list of Azure locations where the container registry should be geo-replicated.

Reviewer: We need to quote the attribute so that the document renderer could render it differently comparing to normal text
```suggestion
* `georeplication_locations` - (Optional / **Deprecated in favor of `georeplications`**) A list of Azure locations where the container registry should be geo-replicated.
```

> ~> **NOTE:** The `georeplication_locations` list cannot contain the location where the Container Registry exists.

Reviewer: COuld we also add the deprecation information to follow the `Optional`? Like this:

* `georeplication_locations` - (Optional / **Deprecated in favor of `georeplications`**) A list of Azure locations where the container registry should be geo-replicated.

Reviewer 2: also can just remove it, but either works

~> **NOTE:** `quarantine_policy_enabled`, `retention_policy` and `trust_policy` are only supported on resources with the `Premium` SKU.

`georeplications` supports the following:

* `location` - (Required) A location where the container registry should be geo-replicated.

> * `tags` - (Optional) A mapping of tags to assign to the container registry replication resource.

Reviewer 2: phrasing
```suggestion
* `tags` - (Optional) A mapping of tags to assign to this replication location.
```
